src/components/grid-aware/ImageHeader/ImageHeader.module.css

```diff
@@ -0,0 +1,139 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-top: 30px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaText {
+  grid-column: 2 / span 6;
+  grid-row: 1;
+}
+
+.gridAreaImage1 {
+  grid-column: 9 / span 4;
+  grid-row: 1;
+}
+
+.gridAreaImage2 {
+  grid-column: 10 / span 3;
+  grid-row: 1;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaImage2 {
+    text-align: right;
+  }
+}
+
+
+/* Left grid area */
+
+.title {
+  font: var(--font-headline);
+  margin-top: 90px;
+}
+
+.subtitle {
+  font: var(--font-body-large);
+  margin-top: 15px;
+}
+
+.description {
+  font: var(--font-body-medium);
+  margin-top: 20px;
+}
+
+.ctaButtonRow {
+  display: flex;
+  flex-flow: row wrap;
+  margin-top: 30px;
+}
+
+.ctaButtonRowItem:not(:last-child) {
+  margin-right: 40px;
+}
+
+@media (--tablet-and-down) {
+  .title {
+    margin-top: 30px;
+  }
+
+  .ctaButtonRow {
+    flex-flow: column;
+    margin-bottom: 30px;
+    margin-top: 20px;
```

Comment:
As far as I can tell from the designs, this top margin is still 30px in mobile, so I think you can just delete this property and let the desktop value of 30px take effect.